From dfda9563434adfed3190b16bca91bf25bf5466e0 Mon Sep 17 00:00:00 2001
From: Varadarajan Narayanan <quic_varada@quicinc.com>
Date: Mon, 23 Jun 2025 17:02:27 +0530
Subject: [PATCH 505/512] arm64: dts: qcom: ipq9574: Fix 'l2' regulator min/max
 voltages

The min and max voltages on l2 regulator is 850000uV. This was
incorrectly set at 1800000uV earlier and that affected the
stability of the networking subsystem.

Fixes: d5506524d9d9 ("arm64: dts: qcom: ipq9574: Add LDO regulator node")
Signed-off-by: Varadarajan Narayanan <quic_varada@quicinc.com>
---
 arch/arm64/boot/dts/qcom/ipq9574-rdp-common.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/ipq9574-rdp-common.dtsi b/arch/arm64/boot/dts/qcom/ipq9574-rdp-common.dtsi
index 0a61e61fa009..42fe38707cd8 100644
--- a/arch/arm64/boot/dts/qcom/ipq9574-rdp-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq9574-rdp-common.dtsi
@@ -250,8 +250,8 @@
 		};
 
 		mp5496_l2: l2 {
-			regulator-min-microvolt = <1800000>;
-			regulator-max-microvolt = <1800000>;
+			regulator-min-microvolt = <850000>;
+			regulator-max-microvolt = <850000>;
 			regulator-always-on;
 			regulator-boot-on;
 		};
-- 
2.34.1

